Library/Homebrew/formula.rb

```diff
@@ -922,6 +897,33 @@ def plist_startup
     self.class.plist_startup
   end
 
+  # This method can be used to provide a plist.
+  # @see https://www.unix.com/man-page/all/5/plist/ Apple's plist(5) man page
+  # <pre>plist do
+  #  {
+  #    Label: "#{plist_name}.override",
+  #    ProgramArguments: ["#{opt_bin}/example", "--do-this"],
+  #    RunAtLoad: true,
+  #    KeepAlive: true,
+  #    StandardErrorPath: "/dev/null",
+  #    StandardOutPath: "/dev/null",
+  #  }
+  # end</pre>
+  def plist
+    plists = self.class.plist.map do |block|
+      plist = instance_eval(&block)
```

Comment:
`plist_hash = ` means you don't end up with the confusing `plist.to_plist` below.  I'm also wondering if there's a new method or other variables that could be used somewhere called `plist_hashes` to clarify that the only stored block plists are hashes.